### ISSUE TITLE
Support null values

### ISSUE THREAD

**Issue by marcojetson** [opensoft/doctrine-postgres-types#12]
When providing null values for array types

```
Common\Error\Exception\Warning: implode(): Invalid arguments passed
/vagrant/vendor/opensoft/doctrine-postgres-types/src/Doctrine/DBAL/PostgresTypes/AbstractArrayType.php (53)
```

**Comment by rybakit** [comment]
Should be fixed in https://github.com/opensoft/doctrine-postgres-types/pull/16.

**Comment by richardfullmer** [comment]
Fixed by #16
